File: python/tests/test_1.py
from python.src.code_1 import Solution
from python.tests.python_deps.boilerplate import BoilerTest


def input_fetcher(case):
    return case["input"]["nums"], case["input"]["target"]


def test():
    test = BoilerTest(1, input_fetcher)
    solution = Solution().twoSum
    test.run_tests(solution)

File: cpp/test/test_1.cpp
#include "../src/code_1.cpp"
#include "cpp_deps/boilerplate.hpp"

using namespace std;

void test(Solution& sol, const json& input, const json& output) {
    vector<int> nums = input["nums"].get<vector<int>>();
    int target = input["target"];
    vector<int> expected = output.get<vector<int>>();
    vector<int> result = sol.twoSum(nums, target);
    sort(result.begin(), result.end());
    CHECK_EQ(result, expected);
}

TEST_CASE("") {
    TEST(1);
}